### ISSUE TITLE
Encoding issue with <b-pagination> since RC12

### ISSUE THREAD

**Issue by billy1380** [bootstrap-vue/bootstrap-vue#3179]
### Describe the bug

There are some artifacts appearing in the paginator since rc12, also tested in rc19 and it seems to be there too.
![image](https://user-images.githubusercontent.com/1329727/56652434-4a037400-6683-11e9-82eb-86d6ac486621.png)

### Expected behavior

![image](https://user-images.githubusercontent.com/1329727/56652556-8cc54c00-6683-11e9-9e92-21c08f192b24.png)

### Versions

**Libraries:**

- BootstrapVue: 2.0.0-rc.12
- Bootstrap: 4.3.1
- Vue: 2.5.17

**Environment:**

- OS: Linux, Windows
- Browser: Chrome, Firefox
- Version: 70

**Comment by tmorehouse** [comment]
Could you provide a minimal reproduction which produces the issue? perhaps in JSFiddle, Codepen, or code sandbox? (tip, you can export from the dos playground to get a base fiddle/pen created)

**Comment by billy1380** [comment]
@tmorehouse I will give it a go and put a link here or an update if I cannot.

**Comment by tmorehouse** [comment]
Is your document specifying the UTF-8 character set in the head?

```html
<head>
  <meta charset="utf-8">
</head>
```

**Comment by billy1380** [comment]
@tmorehouse yes I have a utf-8 charset meta tag in the header.

**Comment by jackmu95** [comment]
@billy1380 Do you see the same error in the docs?
https://bootstrap-vue.js.org/docs/components/pagination

**Comment by billy1380** [comment]
@jackmu95 actually no... I don't. I also don't actually get it on our app until we build and deploy it. So I cannot see it on dev. Using rc.11 they are not there and then update to rc.12-19 the artifacts appear. I have updated and reverted a couple of times just to prove that it is the upgrade that is causing it.

**Comment by tmorehouse** [comment]
Would you be able to paste a copy of the rendered output (for the pagination component) here?

**Comment by billy1380** [comment]
@tmorehouse I will upgrade do another build and paste the markup first thing tomorrow morning.

**Comment by tmorehouse** [comment]
PR #3186 may address this issue

**Comment by billy1380** [comment]
@tmorehouse 
Generated html below.

``` HTML
<ul role="menubar" aria-disabled="false" aria-label="Pagination" class="pagination b-pagination pagination-md"><li role="none presentation" aria-hidden="true" class="page-item disabled"><span role="menuitem" aria-label="Go to first page" aria-disabled="true" class="page-link">Â«</span></li><li role="none presentation" aria-hidden="true" class="page-item disabled"><span role="menuitem" aria-label="Go to previous page" aria-disabled="true" class="page-link">â€¹</span></li><!----><li role="none presentation" class="page-item active"><a role="menuitemradio" aria-label="Go to page 1" aria-checked="true" aria-posinset="1" aria-setsize="1" tabindex="0" target="_self" href="#" class="page-link">1</a></li><!----><li role="none presentation" aria-hidden="true" class="page-item disabled"><span role="menuitem" aria-label="Go to next page" aria-disabled="true" class="page-link">â€º</span></li><li role="none presentation" aria-hidden="true" class="page-item disabled"><span role="menuitem" aria-label="Go to last page" aria-disabled="true" class="page-link">Â»</span></li></ul>
```
I also double checked the head tag and definitely got utf-8 there

I also installed charset chrome extension and for some reason it thinks the page encoding is Western (Windows-1252). If I force the encoding to utf-8 the artifacts disappear as expected.

**Comment by tmorehouse** [comment]
Might be an issue with the response headers from the web server (they usually include a charset header as well).

Can you inspect the request/response headers in the network tab of chrome devtools?

**Comment by billy1380** [comment]
Just checked all the application/javascript content type headers and none of them have encoding specified.

**Comment by tmorehouse** [comment]
@billy1380 Could you try setting the prev/next/first/last button props to the following to see if hte issue persists:

```html
  first-text="\u00AB"
  prev-text="\u2039"
  next-text="\u203A"
  last-text="\u00BB"
```

These are the unicode escape sequences for the respective characters used as the button content.

**Comment by billy1380** [comment]
@tmorehouse that worked great... I did a test on one of our pages...
```  <b-pagination
        :total-rows="pager.totalCount"
        :per-page="pager.count"
        v-model="pager.current"
        :first-text="'\u00AB'"
        :prev-text="'\u2039'"
        :next-text="'\u203A'"
        :last-text="'\u00BB'"
      />
```
That one works well where as other pages that don't have the attributes still showed the artifacts.

**Comment by jackmu95** [comment]
Closed by #3186.

**Comment by U3R1YXJ0** [comment]
This issue persists in rc25. I haven't been able to work out why, as I can see the definition of the strings has changed in pagination.js.
